docs/console/tutorials/create-extension.md

```diff
@@ -4,204 +4,267 @@ title: Getting started
+:::
+
+Finally, deploy the project and check that the services are up and running to be sure that the APIs are exposed and ready to communicate with the extension.
+
+### 2. Create the extension
+
+Once that all requisites are satisfied and the API is ready to be called, you can create the extension.
+You can do that by accessing to your company page and, in the sidebar menu, clicking the `Extensions` option.
+
+![Create your first extension](./img/extensions-create.png)
+
+From this page you will see the list of existing extensions, if there are, otherwise you'll be see a screen that suggest you to create your first extension.
+In any way, a button _Add extension_ is available and, after the click, a menu will show up to allow you to choose which type of extension you want to create.
+
+In this case, you will go with _Create with Composer_.
+
+In the modal that will show up, you can create the extension by choosing:
+
+- the name of the extension (e.g. _My Jira Extension_)
+- the _Template_, which is a starting template for your composer page (you can start with a table by selecting _Table: base table_)
+- the _Destination Area_, in which section of the console the extension will be visible (select _Company Overview_ to have it viewable on the Company page)
+- the _Category_, which is the group where the extension will be located (e.g. select _Administration_)
+
+The _Visibility_ should be automatically selected to _Whole Company_ to indicate that the extension is visible for those who will access to the Company page.
+
+Finally, click on _Add extension_ to create your first extension and to be automatically redirected to its detail page.
+
+![Details of the extension](./img/extensions-detail.png)
+
+The page will recap some information regarding the new extension, such is name, its identifier and where it will be shown.
+Also, you might notice that a new voice is available in the page sidebar, and it has the same name of the extension.
+
+This menu is in fact the extension you have just created. You can access to it, to show an empty table with a loader that, unfortunately, will not load anything.
+This happens because, right now, the table have no indication on where to find the data to show.
+But you are going to fix that.
 
-where in the file `my-extension.json` there are the info to register:
+### 3. Configure the Outbound calls
+
+First of all, you need to configure the extension to ensure that can execute server request to the API connected to the _Integration Connector Agent_, created [in the project that we previously configured](#1-prepare-the-integration-connector-agent-in-your-project).
+
+To do that, you need to return to the page with the detail of the extension.
+If you are not there anymore, you can return there clicking to the _Extensions_ voice in the sidebar, then clicking on the name of the extension you just created.
+
+When you are there, click on the tab _Outbound calls_.
```

Comment:
to make it easier to read, I would include a screenshot of the data to be entered and the interface.

Reply:
I've included one of the _Outbound calls_ page with the initial placeholder.